ad7768-1fmcz/ad77681.c

```diff
@@ -118,9 +124,24 @@ int32_t ad77681_spi_reg_read(struct ad77681_dev *dev,
 	if (ret < 0)
 		return ret;
 
+	if (dev->crc_sel != AD77681_NO_CRC)														// XOR or CRC checksum for read transactions
```

Comment:
Comments should be like this: `/*  XOR or CRC checksum for read transactions */`